strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactControllerTest.java

```diff
@@ -507,6 +508,7 @@ public void testRemoteProxyGroup()
     }
 
     @Test
+    @Disabled
```

Comment:
...And this?

Reply:
Yes, it is giving 400 error. So disabled it

Reply:
Yes, this is related to #1960 and I've advised @madhukarbharti to temporarily disable this method. On 17th and 18th of November, these tests will be failing 100% of the time. We should revert this before merging (if they've lifted the ban) and in due time we should also look into #1960.

Reply:
Okay, thanks for the clarification! :)